### PR TITLE
Deply app digtal ocean

### PR DESCRIPTION
**Pull request by Yazan10x**

### CODE REVIEW

**Review by AsserIsmail — APPROVED** [review]
Intersting

File: models/usec_secrets.py [hunk 1]
```diff
@@ -1,32 +1,50 @@
...
+        _DB_PASSWORD = os.environ.get('DB_API_KEY')
 
-    # _USEC_DB_ADDRESS = 'mongodb+srv://backend:NSW7mvZOheSPHsoS@cluster0.vg0d1j9.mongodb.net/test'
-    _USEC_DB_ADDRESS = f'mongodb+srv://{_DB_USERNAME}:{_DB_PASSWORD}@{_DB_HOST}'
+        _USEC_DB_ADDRESS = f'mongodb+srv://{_DB_USERNAME}:{_DB_PASSWORD}@{_DB_HOST}'
+        _USEC_DB_NAME = 'USEC_DB_TESTING'
 
-    if _PRODUCTION:
-        _USEC_DB_NAME = 'USEC_DB'
     else:
```
Comment (ShaarifAliSyed) [inline models/usec_secrets.py:21]: Before the else, you removed the if statement, so with that being gone, I think you remove the else as well and keep the with open(...

**Review by ShaarifAliSyed — CHANGES_REQUESTED** [review]

File: models/usec_secrets.py (hunk 1 above)
Comment (Yazan10x) [reply models/usec_secrets.py:21]: I removed it on line 19 and added it at line 13

Comment (ShaarifAliSyed) [reply models/usec_secrets.py:21]: Great

**Pull request merged**